.gitignore

```diff
@@ -0,0 +1 @@
+/build*
```

Comment:
Please don't add a gitignore to handle your buildsystem. I think ignored files that solely depend on your setup should be specified in a user gitignore or the private one in .git.